server/src/main/java/org/jboss/as/server/deployment/Phase.java

```diff
@@ -613,6 +613,7 @@ public Phase next() {
     public static final int POST_MODULE_UNDERTOW_SERVLET_CONTAINER_DEPENDENCY       = 0x2F1A;
 
     public static final int POST_MODULE_EE_CONCURRENT_CONTEXT           = 0x3000;
+    public static final int POST_MODULE_EE_STARTUP_COUNTDOWN            = 0x3001;
```

Comment:
@spyrkob Does this need to immediately follow POST_MODULE_EE_CONCURRENT_CONTEXT or just be after and before POST_MODULE_BATCH_ENVIRONMENT?

If the latter, 0x3080 is better.

Reply:
Updated to 0x3080. 

Timing-wise it just need to happen after POST_MODULE_EE_CONCURRENT_CONTEXT and before INSTALL phase